### PR TITLE
Allow to specify different units for lvg pesize (physical extent size)

### PR DESCRIPTION
**Pull request by zeten30**
##### SUMMARY
Allow to specify different units for lvg pesize (physical extent size)

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
modules/system/lvg.py

##### ANSIBLE VERSION
```
ansible 2.4.0.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/brq/mzink/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.13 (default, Aug 16 2017, 12:56:26) [GCC 7.1.1 20170802 (Red Hat 7.1.1-7)]
```

### CODE REVIEW

File: lib/ansible/modules/system/lvg.py [hunk 1]
```diff
@@ -3,7 +3,8 @@
 
 # (c) 2013, Alexander Bulimov <lazywolf0@gmail.com>
 # based on lvol module by Jeroen Hoekx <jeroen.hoekx@dsquare.be>
-# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
```
Comment (alikins) [inline lib/ansible/modules/system/lvg.py]: This is unrelated to the rest of the pr. And this should be kept consistent with other code.

File: lib/ansible/modules/system/lvg.py [hunk 2]
```diff
@@ -124,12 +136,13 @@ def parse_pvs(module, data):
...
     return pvs
 
+
 def main():
     module = AnsibleModule(
-        argument_spec = dict(
+        argument_spec=dict(
             vg=dict(required=True),
             pvs=dict(type='list'),
-            pesize=dict(type='int', default=4),
```
Comment (alikins) [inline lib/ansible/modules/system/lvg.py]: If possible, split the pep8 changes to a separate pr.

**Review by alikins — CHANGES_REQUESTED** [review]
Separate the pep8 changes into a different pr.

**Comment by zeten30** [comment]
@alikins  I've removed flake8/pep8 code optimizations. I'll create another PR later. Thanks for review.

**Comment by zeten30** [comment]
This is just a typo fix.

**Comment by zeten30** [comment]
@alikins  Am I still missing something?

**Comment by gundalow** [comment]
As agreed in 2017-12-07 Core meeting, we've done bulk pep8 fixes across all modules in `devel`.

This was done in https://github.com/ansible/ansible/pull/33682 and https://github.com/ansible/ansible/pull/33690 which have been merged into `devel`. because of this you will need to rebase your PR. Thanks in advance.
